From 69d8465e3f252da8ed4f16304b35459c96801521 Mon Sep 17 00:00:00 2001
From: rigaz29 <ryanbagas27@gmail.com>
Date: Fri, 7 Aug 2026 01:04:19 +0000
Subject: [PATCH 13/13] Revert "kernel: add `pahole` dependency"

This reverts the pahole requirement for legacy kernel builds (UL 03a14c2b),
restored during M3 conflict resolution (keep-both loop dropped it because the
3-way merge resolved empty).

Change-Id: I9074a97c4a37c89e496df9e0eb972497c67c19a2
---
 config/BoardConfigKernel.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/config/BoardConfigKernel.mk b/config/BoardConfigKernel.mk
index 4c2a268d..ca690d4b 100644
--- a/config/BoardConfigKernel.mk
+++ b/config/BoardConfigKernel.mk
@@ -222,8 +222,6 @@ KERNEL_MAKE_FLAGS += YACC=$(BUILD_TOP)/prebuilts/build-tools/$(HOST_PREBUILT_TAG
 KERNEL_MAKE_FLAGS += M4=$(BUILD_TOP)/prebuilts/build-tools/$(HOST_PREBUILT_TAG)/bin/m4
 TOOLS_PATH_OVERRIDE += BISON_PKGDATADIR=$(BUILD_TOP)/prebuilts/build-tools/common/bison
 
-# Since Linux 5.10, pahole is required
-KERNEL_MAKE_FLAGS += PAHOLE=$(BUILD_TOP)/prebuilts/kernel-build-tools/linux-x86/bin/pahole
 
 # Set the out dir for the kernel's O= arg
 # This needs to be an absolute path, so only set this if the standard out dir isn't used
-- 
2.43.0

